docs/src/overview.md

```diff
@@ -0,0 +1,112 @@
+```@meta
+DocTestSetup = :(using Ferrite)
+```
+
+# Documentation of examples
+
+On this page, you find a documentation for each example currently available for `Ferrite`. The collection contains programs displaying different aspects and techniques of the `Ferrite` toolbox. 
+The programs are sorted as
+
+* A complete list shortly summarizing the topic, what they teach and which keywords related to the task they contain
+* Grouped by topic 
+
+## List of examples (chronological order)
```

Comment:
```suggestion
## List of examples (sequential order)
```
My intention was to create a list with the same order as on the sidebar of the homepage. I changed the desciption to match that, though it may be possible examples later in the list were added earlier (in time).